packages/frontend/src/components/pages/search-page/SearchPage.jsx

```diff
@@ -77,7 +65,7 @@ const SearchPage = () => {
           </Form>
         )}
       </Formik>
-      <OrderTable rows={rows} />
+      {rows ? <OrderTable rows={rows} /> : null}
```

Comment:
эту конструкцию можно записать немного проще:
```suggestion
      {rows && <OrderTable rows={rows} />}
```
[Здесь можно подробнее почитать про это](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)